onnxruntime/core/graph/graph.cc

```diff
@@ -3067,66 +3067,6 @@ ONNX_NAMESPACE::GraphProto Graph::ToGraphProto() const {
   return result;
 }
 
-ONNX_NAMESPACE::GraphProto Graph::ToGraphProtoWithExternalInitializers(const std::string& external_file_name,
-                                                                       size_t initializer_size_threshold) const {
-  GraphProto result;
-  ToGraphProtoInternal(result);
-  const auto& model_path = ModelPath();
-
-  std::ofstream external_stream(external_file_name, std::ofstream::out | std::ofstream::binary);
-  ORT_ENFORCE(external_stream.is_open());
-  int64_t external_offset = 0;
-
-  // Add the initializers to the result graph.
-  const auto sparse_end = sparse_tensor_names_.end();
-  for (const auto& initializer : graph_proto_->initializer()) {
```

Comment:
graph_proto_->initializer() probably has been destroyed if the model has been used for creating a session.  We want to support a user scenario that:

1. Load a model into memory
2. Run onnx runtime optimizers
3. Save the optimized model to disk.

Then we need to do it at a different level.